### ISSUE TITLE
Documentation: Tag variable is not properly documented in example, needs amending.

### ISSUE THREAD

**Issue by williamfalconeruk** [diodonfrost/terraform-aws-lambda-scheduler-stop-start#16]
##### SUMMARY
readme.MD

##### ISSUE TYPE
- Documentation update 

##### TERRAFORM VERSION

```paste below
Terraform v0.13.6
+ provider registry.terraform.io/hashicorp/archive v2.1.0
+ provider registry.terraform.io/hashicorp/aws v2.70.0
+ provider registry.terraform.io/hashicorp/local v2.1.0

Your version of Terraform is out of date! The latest version
is 0.14.7. You can update by downloading from https://www.terraform.io/downloads.html
```

##### STEPS TO REPRODUCE

run  plan or apply using example on Readme.MD

```paste below
terraform apply
```

##### EXPECTED RESULTS
plan and apply to succeed

##### ACTUAL RESULTS
The error message below.

```paste below
Error: Unsupported argument

  on rds_hibernation.tf line 10, in module "start_rds_instances":
  10:   scheduler_tag = {

An argument named "scheduler_tag" is not expected here.
```
the solution is to align examples in the docs with inputs  and outputs etc. e.g `scheduler_tag` should be `resources_tag`
I attempted to create a branch but do not have push permissions to update the docs in a PR.

**Comment by diodonfrost** [comment]
Hello, it should be fixed in the version [3.1.3](https://github.com/diodonfrost/terraform-aws-lambda-scheduler-stop-start/releases/tag/3.1.3)

**Comment by williamfalconeruk** [comment]
thankyou for this, also for making it backwards compatible too.
